README.md

```diff
@@ -779,6 +779,13 @@ Table of Contents
   * [jare.io](http://www.jare.io) — CDN for images. Uses AWS CloudFront
   * [jsdelivr.com](https://www.jsdelivr.com/) — CDN of OSS (JS, CSS, fonts) for developers and webmasters, accepts PRs to add more
   * [Microsoft Ajax](https://docs.microsoft.com/en-us/aspnet/ajax/cdn/overview) — The Microsoft Ajax CDN hosts popular third-party JavaScript libraries such as jQuery and enables you to easily add them to your Web application
+  * [MerlinCDN](https://www.merlincdn.com/)
```

Comment:
This can be one line, please collapse so its easier to sort.  We only split really massive things out into multiple lines.

Reply:
Also I think this just simply isn't suitable for this list, We dont take things like $10 credit, see the top of the file for criteria.